File: FrameIncam.Client/ng/domains/master/geo/masterGeoLevel.js
﻿(function ($, ng, debugMode, undefined) {
    "use strict";

    // --------------------------------------------------------------------------------
    // Dependencies
    // --------------------------------------------------------------------------------
    var module = ng.module("app");

    // --------------------------------------------------------------------------------
    // Model
    // --------------------------------------------------------------------------------
    module.run([
        "services.utils",
        "services.models",
        function (
            p_utils,
            p_models
        ) {
            // --------------------------------------------------------------------------------
            // Namespace
            // --------------------------------------------------------------------------------
            var fincam = p_models.namespace("fincam");
            var geo = p_models.namespace("fincam.master.geo");

            // --------------------------------------------------------------------------------
            // Model
            // --------------------------------------------------------------------------------
            p_models.class({
                namespace: geo,
                name: "masterGeoLevel",
                base: {
                    namespace: fincam,
                    name: "model"
                },
                properties: {
                    id:  "int",
                    geoLevelCode: "int",
                    geoLevelName:  "string",
                    geoLevelIsactive: "int"
                },
                prototype: new function () {
                    var self = this;
                    return self;
                }
            });
        }
    ]);

})(jQuery, angular, document.children[0].hasAttribute("debug"));

File: FrameIncam.Domains/Models/Master/Geo/MasterGeoLevel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace FrameIncam.Domains.Models.Master.Geo
{
    [Table("master_geo_level")]
    public class MasterGeoLevel : Model
    {
        [Column("Geo_Level_Code")]
        public int? GeoLevelCode { get; set; }
        [Column("Geo_Level_Name")]
        public string GeoLevelName { get; set; }
        [Column("Geo_Level_Isactive")]
        public int GeoLevelIsactive { get; set; }
    }
}
